system/Cache/Handlers/DummyHandler.php

```diff
@@ -88,6 +88,20 @@ public function delete(string $key)
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Deletes items from the cache store matching a given pattern.
+	 *
+	 * @param string $pattern Cache items glob like pattern
+	 *
+	 * @return boolean
+	 */
+	public function deleteMatching(string $pattern)
+	{
+		return true;
```

Comment:
Why is this returning true while others are false?

Reply:
The method returns true for success. The dummy handler always returns true so it doesn't break the code.

For memcached and wincache, as there are limitations, it doesn't do anything but would break if used in a condition.
It is so that users wouldn't use it. But, now that I think of it, maybe it is wiser to throw an exception stating that the method isn't implemented for those specific handlers.

Reply:
I think we should go for throwing exceptions if not implemented.